src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,208 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.ManufacturerService;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    @Inject
+    private ManufacturerService manufacturerService;
+
+    @Override
+    public Car create(Car car) {
+        String createCarQuery = "INSERT INTO cars (model, manufacturer_id) VALUES (?, ?)";
+        String createDriversRelationQuery =
+                "INSERT INTO cars_drivers(driver_id, car_id) VALUES (?, ?);";
```

Comment:
I think it's better to move car-drivers relation creation in separate private method. If in the future we have new relationships for car and we need to add them to the create method, then this method will come out too large. And we could reuse it in update method. (same in update method)